src/app/components/shared/nav/nav.component.html

```diff
@@ -0,0 +1,13 @@
+<nav class="main-nav">
+    <div class="row">
```

Comment:
nav should not contain img or other tags. Only main navigation items

Reply:
I have worked on this and will commit the code shortly.